mirdata/utils.py

```diff
@@ -233,9 +234,16 @@ def __init__(self, index_file, metadata_load_fn=None):
         self._metadata = None
         self.index_file = index_file
         self.metadata_load_fn = metadata_load_fn
+        self.download_path = download_path
 
     @cached_property
     def index(self):
+        if self.download_path is not None:
```

Comment:
this should be tested a bit differently: if the index does not exist locally and if self.download_path is not None

Reply:
+1

Reply:
Yes!! Tomorrow I wil do the tests!